classes/Cart.php

```diff
@@ -1025,6 +1025,10 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
         $pa_implode = [];
         $separator = Configuration::get('PS_ATTRIBUTE_ANCHOR_SEPARATOR');
 
+        if ($separator === '-') {
```

Comment:
Don't we have to add a space after only the dash separator ?

Reply:
A space after the separator is already added, regardless of the separator type (comma or dash) a few lines below.
To my knowledge there is only two separator, comma (,) and dash (-).
In this case, only the dash needs a extra space before. That's why I add a space only if the separator is the dash

Reply:
This `PS_ATTRIBUTE_ANCHOR_SEPARATOR` configuration is used many times, is space is needed elsewhere?

Reply:
TBH I don't know. I tried to take a look but I am not sure. Obviously it would be easier to change it on a global scale.

Reply:
After discussion with @LouiseBonnard, you could remove the condition on `-` and add a space after any separator.

Reply:
I think you misread the whole thing. There is already a space added **after** any separator on lines 1061-1062

https://github.com/PrestaShop/PrestaShop/blob/13f80eadd7d2f4d5f669a4f5f1e64701e5ad4241/classes/Cart.php#L1061-L1062

Problem here I am trying to solve is to also add a space **before** the dash (but not before the comma).